Python/optimizer_bytecodes.c

```diff
@@ -789,21 +789,23 @@ dummy_func(void) {
         }
     }
 
-    op(_YIELD_VALUE, (unused -- res)) {
-        res = sym_new_unknown(ctx);
+    op(_YIELD_VALUE, (retval -- value)) {
+        value = sym_new_unknown(ctx);
     }
 
-    op(_FOR_ITER_GEN_FRAME, ( -- )) {
+    op(_FOR_ITER_GEN_FRAME, (iter -- iter, gen_frame: _Py_UOpsAbstractFrame*)) {
+        gen_frame = NULL;
```

Comment:
I am not entirely sure about this. Should I be setting the frame to `NULL` here? I saw a similar pattern in another uop.

Reply:
The setting to NULL should be just to please the cases generator and tell it that `gen_frame` is set to something. I think if you remove it, the cases generator will error.

Reply:
> I think if you remove it, the cases generator will error.

Yup, you're right. I just wanted to make sure that setting it to `NULL` wouldn't blow up somewhere else :smile: Thanks!